data/publiccloud/terraform/azure.tf

```diff
@@ -56,6 +56,10 @@ variable "create-extra-disk" {
 }
 
 variable "storage-account" {
+    # Note: Don't delete the default value!!!
+    # Not all of our `terraform destroy` calls pass this variable and neither is it necessary.
+    # However removing the default value might cause `terraform destroy` to fail in corner cases,
+    # resulting effectively in leaking resources due to failed cleanups.
     default="openqa"
```

Comment:
and why do we keep `openqa` as default keeping in mind that we already switched majority of our tests to run in new account ?

Reply:
incidents still run in old ones.